audinterface/core/utils.py

```diff
@@ -221,28 +221,34 @@ def signal_index(
     Raises:
         ValueError: if ``start`` and ``ends`` differ in size
 
-    Example:
+    Examples:
+
         >>> signal_index(0, 1.1)
         MultiIndex([('0 days', '0 days 00:00:01.100000')],
                    names=['start', 'end'])
+
```

Comment:
Why the blank line here?

Reply:
To split independent examples into different blocks:

![image](https://user-images.githubusercontent.com/173624/208077156-0640e39d-3eff-4f05-8818-edbfd867efb1.png)

Reply:
Mhh, I'm not convinced :)

Reply:
Would you then stick to the compact format all the time, or split it up when we write explanatory text inbetween like we do here:

![image](https://user-images.githubusercontent.com/173624/208079482-1dd7c043-ff10-430b-bfd3-0faf932b48ca.png)

Reply:
yes, exactly

Reply:
I mean yes, only break the compact form when we write explanatory text :)

Reply:
As discussed below, we do no longer split up the examples.